plugin.py

```diff
@@ -86,10 +86,28 @@ def install_or_update(cls) -> None:
     def markdown_language_id_to_st_syntax_map(cls) -> Optional["MarkdownLangMap"]:
         return {"python": (("python", "py"), ("LSP-pyright/syntaxes/pyright",))}
 
+    def on_server_response_async(self, method: str, response: Response) -> None:
+        if method == "textDocument/hover" and isinstance(response.result, dict):
+            contents = response.result.get("contents")
+            if not isinstance(contents, dict) or not contents.get("kind") == "markdown":
+                return
+            response.result["contents"]["value"] = self.patch_markdown_content(contents["value"])
+
     # -------------- #
     # custom methods #
     # -------------- #
 
+    def patch_markdown_content(self, content: str) -> str:
```

Comment:
One other place where it's common to look at documentation is the completion documentation. That would be a `completionItem/resolve` request. Any plans to handle that too?